src/jit/compiler.cpp

```diff
@@ -1007,6 +1008,7 @@ var_types Compiler::getReturnTypeForStruct(CORINFO_CLASS_HANDLE clsHnd,
         // and also examine the clsHnd to see if it is an HFA of count one
         useType = getPrimitiveTypeForStruct(structSize, clsHnd);
     }
+#endif
```

Comment:
I think so. https://github.com/dotnet/coreclr/issues/10027#issuecomment-285309609 (`IsUnmanagedValueTypeReturnedByRef` always returns true) implies that. Anyway the change here may affect the managed code calling convention, even if it works fine. I should look at it again. Thank you!